sympy/stats/crv_types.py

```diff
@@ -1765,6 +1785,12 @@ def Kumaraswamy(name, a, b):
 class LaplaceDistribution(SingleContinuousDistribution):
     _argnames = ('mu', 'b')
 
+    set = Interval(-oo, oo)
+
+    @staticmethod
+    def check(mu, b):
+        _value_check(b > 0, "Scale parameter b must be positive.")
```

Comment:
What about `_value_check(mu.is_real, "Location parameter mu should be real")`?

Reply:
Done. Thanks!